xml/release-notes.xml

```diff
@@ -334,6 +334,27 @@
     <package>libzypp</package> versions older than 12.25.8:
    </para>
    <screen>zypper dup --allow-vendor-change --force-resolution</screen>
+
+   <para>
+    &opensuseleap; 15.3 supplies all required rpm veryfication keys including the &sles;
+    as part of <package>openSUSE-build-key</package>.
+    All keys are also newly available in the oss repository.
+
+    <package>libzypp-17.25.11</package> imports all keys automatically.
+    However, we recommend following steps in case you need to import keys manually.
```

Comment:
In case that user does not have imported the key which signs the repodata ... keys which are just used to sign packages will be auto-imported